samcli/lib/providers/provider.py

```diff
@@ -494,8 +494,21 @@ def cors_to_headers(cors: Optional["Cors"]) -> Dict[str, Union[int, str]]:
         """
         if not cors:
             return {}
+
+        # cors.allow_origin can be either a single origin or comma separated list of origins
```

Comment:
I'm not 100% sure this is the right place to do matching. Would appreciate guidance if this has to go somewhere else.

Reply:
This looks like a good place for this to live :)